### DISCUSSION TITLE
Multiple tables collapse on mobile

### DISCUSSION THREAD

**Discussion by manuelpmgt** [rappasoft/laravel-livewire-tables#1657] [Q&A]
Hi, I cant find the answer to this problem.  I have multiple tables on a page and everything is working fine, the problem is when i have collapsible columns, when I expand a row, the same row on all tables collapse or show at the same time. I have different names on all tables and still doesnt work. Thank you.

**Comment by lrljoe** [comment]
> Hi, I cant find the answer to this problem.  I have multiple tables on a page and everything is working fine, the problem is when i have collapsible columns, when I expand a row, the same row on all tables collapse or show at the same time. I have different names on all tables and still doesnt work. Thank you.

Will take a look, early next week most likely!

I suspect I've missed some uniqueness in terms of identifying the rows correctly

**Reply by manuelpmgt** [reply]
Thank you so much! Do you need me to create an issue? Or is it fine like this?

**Reply by lrljoe** [reply]
If you could please create an Issue, it means I can add it into the tracker so it doesn't get missed.

I was just looking at the collapsing code, and it should be relatively simple to fix
